GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+
+## Decision 
+### Team members
+Team member status may be given to those who have made ongoing contributions to the Fn Project for at least 3 months. This is usually in the form of code improvements and/or notable work on documentation, but organizing events or user support will also be taken into account.
+
+New members may be proposed by any existing member by posting to the Fn Project forums. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
+
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This will also be posted to the Fn Project forums for record-keeping purposes.
+
+If they choose to accept, the following steps are taken:
+  * Team members are added to the [GitHub organization](https://github.com/fnproject) as _Maintainer_.
+  * Team members are added to the [Fn Team group on forums.fnproject.io](https://forums.fnproject.io/groups/FnTeam).
+  * Team members are added to the list of team members in this document.
+  * New team members are announced on the Fn Project forums by an existing team member.
+
+Team members may retire at any time by posting to the Fn Project forums and removing themselves from the [Fn Team group](https://forums.fnproject.io/groups/FnTeam).
+
+Team members can be removed by [supermajority vote](#supermajority-vote) in the [private Fn Team category in the Fn Project forums](https://forums.fnproject.io/c/fn-team). For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+The current team members are:
+  * dssdfa
+  * hjklhj
+  * werqrr
+  * uioppp
+  * vbnmvn
+
+### Maintainers
+Maintainers lead one or more project(s) or parts thereof and serve as a point of conflict resolution amongst the contributors to this project. Ideally, maintainers are also team members, but exceptions are possible for suitable maintainers that, for whatever reason, are not yet team members.
+
+Changes in maintainership have to be announced on the Fn Project forums . They are decided by [lazy consensus](#lazy-consensus) and formalized by changing the `MAINTAINERS.md` file of the respective repository.
+
+Maintainers are granted commit rights to all projects in the [GitHub orgaization](https://github.com/fnproject).
+
+A maintainer may resign by messaging the [Fn Team on the forums](https://forums.fnproject.io/groups/FnTeam) . A maintainer with no project activity for a year is considered to have resigned. Maintainers that wish to resign are encouraged to propose another team member to take over the project.
+
+A project may have multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
```

Comment:
> clearly agreed upon between them

...and documented in `maintainers.md` so that all team members know how it's split, perhaps?